packages/contracts-bridge/contracts/test/NFTAccountant.t.sol

```diff
@@ -30,18 +30,77 @@ contract NFTAccountantTest is Test {
         accountant = new NFTRecoveryAccountantHarness(address(this), recipient);
         accountant.initialize();
         // setup test vars
-        user = vm.addr(90210);
+        defaultUser = vm.addr(90210);
```

Comment:
can we pre-emptively separate fixtures from mock state in prep for porting these to fork test?